src/Microsoft.AspNet.Mvc.Razor/RazorView.cs

```diff
@@ -42,6 +42,9 @@ public class RazorView : IView
             IsPartial = isPartial;
         }
 
+        /// <inheritdoc />
+        public string Path => RazorPage.Path;
```

Comment:
The generated code looks like:
```C#
public string Path
{
   get { return RazorPage.Path; }
}
```
There are no delegates \ backing field in the generated code .